packages/form/addon/lib/field.js

```diff
@@ -423,12 +424,16 @@ export default class Field extends Base {
   get jexlContext() {
     const parent = this.fieldset.field?.fieldset.form;
 
+    const rootDocument = this.document.parentDocument ?? this.document;
```

Comment:
But this doesn't work for questions in table, does it? In that case the `parentDocument` will refer to the document where the table is, not the `parentDocument` of the case.

Also, are you sure that the `parentDocument` is set for documents that aren't table rows?

Reply:
As discussed, `parentDocument` is covering questions in table rows, and `document` covers non-table questions.